bazel/repository_locations.bzl

```diff
@@ -1278,7 +1278,6 @@ REPOSITORY_LOCATIONS_SPEC = dict(
             "envoy.filters.network.rbac",
             "envoy.filters.network.wasm",
             "envoy.stat_sinks.wasm",
-            "envoy.rbac.matchers.upstream_ip_port",
```

Comment:
Note for reviewer: initially CI wanted me to add my new extension to the list here due to a transitive dependency on CEL from `rbac:matchers_lib`. Instead of that, I broke it into two bazel components so that pieces that don't actually use CEL don't depend on CEL anymore.

Reply:
Nice!